import_export/formats/base_formats.py

```diff
@@ -220,3 +220,16 @@ def create_dataset(self, in_stream):
             row_values = [cell.value for cell in row]
             dataset.append(row_values)
         return dataset
+
+#: These are the default formats for import and export. Whether they can be
+#: used or not is depending on their implementation in the tablib library.
+DEFAULT_FORMATS = (
+    CSV,
+    XLS,
+    XLSX,
+    TSV,
+    ODS,
+    JSON,
+    YAML,
+    HTML,
+)
```

Comment:
Maybe have this in __init__.py? so we don't have the define at the very end of the file. @bmihelac

Reply:
@manelclos I hate to write in `__init__.py`. In my opinion, it should contains metadata, not real code.